### PR TITLE
Add menu bar to the main window

### LINKED ISSUE #51: Add a menu widget
# 📖 Overview

The menu should have `File`, `Edit`, and `Help` for now. This issue is simply for the widget itself, not any of the menu options or actions

### PR DESCRIPTION
**Pull request by Kangaroux**
Closes #51

### CODE REVIEW

**Comment by Kangaroux** [comment]
The model for the menu is defined in [MenuModel.cs](https://github.com/OpenChartProject/OpenChart/tree/feature/ui-menu/OpenChart/src/UI/Actions/MenuModel.cs)

The [MenuBar](https://github.com/OpenChartProject/OpenChart/blob/feature/ui-menu/OpenChart/src/UI/Widgets/MenuBar.cs) widget binds to the model using the "app" namespace since the actions are added at the Application level, not the Window level.

Finally, actions are added to the app on startup in [InitActions](https://github.com/OpenChartProject/OpenChart/blob/feature/ui-menu/OpenChart/src/Application.cs#L42)

I added a [QuitAction](https://github.com/OpenChartProject/OpenChart/blob/feature/ui-menu/OpenChart/src/UI/Actions/File/QuitAction.cs) just to test it. Hotkeys for the menu aren't working at the moment

image

File: .vscode/launch.json [hunk 1]
```diff
@@ -15,7 +15,10 @@
             "cwd": "${workspaceFolder}",
             // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console
```
Comment (monotron) [inline .vscode/launch.json:16]: Comments are nonstandard in JSON, please consider removing

Comment (Kangaroux) [reply .vscode/launch.json:16]: I'm going to go on record to say that vscode added those, not me lol

Comment (monotron) [reply .vscode/launch.json]: Excuses, excuses! :)

**Pull request merged**